### PR TITLE
Fix turnreveal flipping out of container

### LINKED ISSUE #305: Activity description 'flips' out of container
When hovering from one to another activity on the activities page, the 'mirrored' activity description flips outside the activity container. Fixable by adding `overflow: hidden;` on the .kbkUje class.

### PR DESCRIPTION
**Pull request by Riscky**
fixes #305

### CODE REVIEW

File: src/components/activities/Activity.jsx [hunk 1]
```diff
@@ -132,6 +132,7 @@ const TurnRevealWrapper = styled.div`
 
   .turnreveal-container {
     width: fit-content;
+    overflow: hidden;
```
Comment (jasperro) [inline src/components/activities/Activity.jsx:135]: This seems to break shadows on the left and right of the turnreveal container,

maybe add something like
```css
padding: 5px;
margin: -5px;
```
This does make the content flip a little bit outside of the container, maybe there is a different solution?

**Review by jasperro — CHANGES_REQUESTED** [review]

File: src/components/activities/Activity.jsx (hunk 1 above)
Comment (Riscky) [reply src/components/activities/Activity.jsx]: Thanks for noticing! 

I moved the shadow to the outer element, and made sure it tightly hugs the inner elements.
I had to override the shadow of one of the inner elements to make sure the shadows don't give weird animation artifacts.

Everything looks nice and good to me now, can you give it a last check?

File: src/components/activities/Activity.jsx [hunk 2]
```diff
@@ -116,23 +116,26 @@ const renderInfo = (
...
+   */
+  box-shadow: none !important;
 `;
 
 const TurnRevealWrapper = styled.div`
   display: flex;
   justify-content: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.15), 0 2px 6px rgba(0, 0, 0, 0.2);
+  border-radius: 5px;
+  overflow: hidden;
```
Comment (jasperro) [inline src/components/activities/Activity.jsx:135]: Maybe best to put
```css
width: fit-content;
justify-self: center;
```
here and
```css
display: grid;
```
in the css for the ```<div class="activity">``` element,

this makes it look the way it was before without extra card borders.

**Review by jasperro — CHANGES_REQUESTED** [review]

File: src/components/activities/Activity.jsx (hunk 2 above)
Comment (Riscky) [reply src/components/activities/Activity.jsx:135]: Good catch, I forgot to check smaller screen sizes.

**Review by jasperro — APPROVED** [review]
Looks good to me!

**Pull request merged**